README.md

```diff
@@ -58,14 +58,17 @@ Again, use `5.x-dev` if you want to add a new feature or `<lts-version>.x-dev` i
 Next, install the dependencies:
 
 ```bash
-composer update
+composer update --prefer-install=auto
```

Comment:
I disagree with this change, because it would use `source` for every package with a dev version. The only dev package that we want is `contao/contao` itself, which already uses `source` due to the branch name (e.g. `4.13.x-dev` or `5.x-dev`).